specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+        },
+        "expirationDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Date of authorization expiration."
+        }
+      },
+      "description": "Authorized groundstation."
+    },
+    "OperationResult": {
+      "description": "Operation Result Entity.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "description": "ID of the resource.",
+          "type": "string"
+        },
+        "name": {
+          "readOnly": true,
+          "description": "Name of the resource.",
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "startTime": {
+          "description": "The operation start time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "endTime": {
+          "description": "The operation end time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "percentComplete": {
+          "type": "number",
+          "format": "double",
+          "minimum": 0,
+          "maximum": 100,
+          "description": "Percentage completed.",
+          "readOnly": true
+        },
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {}
+          },
+          "x-ms-identifiers": [],
+          "description": "A list of results when the operation returns multiple results."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/OperationResultProperties"
+        },
+        "error": {
+          "$ref": "#/definitions/OperationResultErrorProperties"
+        }
+      }
+    },
+    "Status": {
+      "readOnly": true,
+      "description": "The status of operation.",
+      "type": "string",
+      "enum": [
+        "Succeeded",
+        "Canceled",
+        "Failed",
+        "Running"
+      ],
+      "x-ms-enum": {
+        "name": "Status",
+        "modelAsString": true
+      }
+    },
+    "OperationResultProperties": {
+      "type": "object",
+      "properties": {},
+      "description": "Operation result properties."
+    },
+    "OperationResultErrorProperties": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The code of the error.",
+          "readOnly": true
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message.",
+          "readOnly": true
+        }
+      },
+      "description": "Operation result error properties."
+    },
```

Comment:
Hopefully you can refactor to be taking advantage of the common-type definitions for these

Reply:
need a discussion on this change. holding off on this one for now

Reply:
we will not be using common-types for this because we have some of our own defined properties which will break things